elasticapm/conf/__init__.py

```diff
@@ -590,6 +590,12 @@ class Config(_ConfigBase):
         validators=[duration_validator],
         type=int,
     )
+    exit_span_min_duration = _ConfigValue(
+        "exit_span_min_duration",
+        default=1,
+        validators=[duration_validator],
```

Comment:
From the [spec](https://github.com/elastic/apm/blob/master/specs/agents/handling-huge-traces/tracing-spans-drop-fast-exit.md):

> The minimum allowed duration for this setting is `1us` (microsecond). Agents may need to add support the `us` unit.

We still need to add support for microseconds, though I don't think it's critical to have in this PR.